frontend/src/components/organism/Projections.tsx

```diff
@@ -108,6 +125,14 @@ export default styled(Projections)`
     }
   }
 
+  .projection-row-in {
+    background: #d6fccf;
```

Comment:
These colours should perhaps be moved to the palette and named `success` and `error` or something similar so we can reuse them later.